dnf5daemon-client/CMakeLists.txt

```diff
@@ -5,6 +5,9 @@ endif()
 set(DNF5DAEMON_CLIENT_BIN dnf5daemon-client)
 file(GLOB_RECURSE DNF5DAEMON_CLIENT_SOURCES *.cpp)
 
+#set gettext domain for translations
+add_definitions(-DGETTEXT_DOMAIN=\"dnf5-daemon-server\")
```

Comment:
This would be better in a separate commit.

Also, is the value correct, shouldn't it be `"dnf5-daemon-client"`?

Reply:
Copypaste error, got it. I put the change on a separate commit. Thanks